runelite-api/src/main/java/net/runelite/api/WorldType.java

```diff
@@ -77,10 +78,13 @@
 	}
 
 	private static final EnumSet<WorldType> PVP_WORLD_TYPES = EnumSet.of(
-		DEADMAN,
```

Comment:
Deadman is still a PVP world type. Why is this removed?

Reply:
Well it can be left, but the deadman check comes before the PVP world check. So if they're in a deadman world, `PVP_WORLD_TYPES` will never be checked.

Reply:
`isPvpWorld` is used outside of the discord plugin, so this would cause adverse effects for other plugins.

Reply:
Fixed.